File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/07_Hotel_Room.js
function hotelRoom(input) {
    let month = input[0];
    let nights = Number(input[1]);

    let studioNight = 0;
    let apartamentNight = 0;

    let studioDiscount = 0;
    let apartamentDiscount = 0;

    if (nights > 14) {
        apartamentDiscount = 0.10;
    }

    if (month == "May" || month == "October") {
        studioNight = 50;
        apartamentNight = 65;
        if (nights > 7 && nights <= 14) {
            studioDiscount = 0.05;
        } else if (nights > 14) {
            studioDiscount = 0.30;
        }
    } else if (month == "June" || month == "September") {
        studioNight = 75.20;
        apartamentNight = 68.70;
        if (nights > 14) {
            studioDiscount = 0.20;
        } 
    } else if (month == "July" || month == "August") {
        studioNight = 76;
        apartamentNight = 77;
    }

    let apartamentTotal = apartamentNight * (1 - apartamentDiscount) * nights;
    let studioTotal = studioNight * (1 - studioDiscount) * nights;

    console.log(`Apartment: ${apartamentTotal.toFixed(2)} lv.`);
    console.log(`Studio: ${studioTotal.toFixed(2)} lv.`);
}

// hotelRoom(["May", "15"]);

File: Python_Courses/Python_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/07_Hotel_Room.py
month = input()  # May, June, July, August, September или October
nights = int(input())

studio_price = 0
apartment_price = 0

if month == "May" or month == "October":
    studio_price = 50
    apartment_price = 65
    if 14 >= nights > 7:
        studio_price *= 0.95
    elif nights > 14:
        studio_price *= 0.70
elif month == "June" or month == "September":
    studio_price = 75.20
    apartment_price = 68.70
    if nights > 14:
        studio_price *= 0.80
elif month == "July" or month == "August":
    studio_price = 76.00
    apartment_price = 77.00

if nights > 14:
    apartment_price *= 0.90

apartment_total_price = nights * apartment_price
studio_total_price = nights * studio_price
print(f"Apartment: {apartment_total_price :.2f} lv.")
print(f"Studio: {studio_total_price :.2f} lv.")
